test/test_helper.rb

```diff
@@ -29,17 +29,15 @@ def capture_output_streams
 
 def with_cloned_fs
   FakeFS do
-    begin
-      FakeFS::FileSystem.clone(PathHelper.project_path)
+    FakeFS::FileSystem.clone(PathHelper.project_path)
 
-      # reek schema is required to init reek
-      FakeFS::FileSystem.clone(PathHelper.reek_schema_path)
+    # reek schema is required to init reek
+    FakeFS::FileSystem.clone(PathHelper.reek_schema_path)
 
-      Dir.chdir(PathHelper.project_path)
-      yield
-    ensure
-      FakeFS::FileSystem.clear
-    end
+    Dir.chdir(PathHelper.project_path)
+    yield
+  ensure
+    FakeFS::FileSystem.clear
   end
```

Comment:
removing the `begin` from the block since th target ruby version in rubocop.yml is set to 2.5 and from ruby 2.5 onwards begin block is seemed redundant as per rubocop rules.
here is that PR when the change was done in rubocop https://github.com/rubocop/rubocop/pull/5175